Test/dafny0/SubsetTypes.dfy

```diff
@@ -42,7 +42,7 @@ module AssignmentToNat {
     requires f.requires(x)  // error
   {
     var y := f(x);  // error
-    g := z => z;  // error
+    // g := z => z;  // type error
```

Comment:
Why is this commented out? Does it no longer produce the expected error?

Reply:
Type resolution errors are tested in SubsetTypesErr.dfy. They are commented out in SubsetTypes.dfy so that that test can proceed to check for verification errors.

Reply:
I see. I had missed the fact that they changed from verification errors to type errors. Then, if they're no longer being used in this file, I think they are better deleted from here (since they are checked in SubsetTypesERR.dfy).

Reply:
Cleaned up these two test files